Client/src/Components/Dialog/index.jsx

```diff
@@ -0,0 +1,80 @@
+import LoadingButton from "@mui/lab/LoadingButton";
+import { Button, Modal, Stack, Typography } from "@mui/material";
+import PropTypes from "prop-types";
+const Dialog = ({ modelTitle, modelDescription,
+    open, onClose, title, confirmationBtnLbl, confirmationBtnOnClick, cancelBtnLbl, cancelBtnOnClick, theme, isLoading, description }) => {
+    return <Modal
+        aria-labelledby={modelTitle}
+        aria-describedby={modelDescription}
+        open={open}
+        onClose={onClose} >
+        <Stack
+            gap={theme.spacing(2)}
+            sx={{
+                position: "absolute",
+                top: "50%",
+                left: "50%",
+                transform: "translate(-50%, -50%)",
+                width: 400,
+                bgcolor: theme.palette.background.main,
+                border: 1,
+                borderColor: theme.palette.border.light,
+                borderRadius: theme.shape.borderRadius,
+                boxShadow: 24,
```

Comment:
In some places we are using theme.shape.boxShadow for this

Reply:
We can open up a new PR to handle that, we should go through the whole frontend and look for inconsistencies like that as we migrated to using the theme only a month or two ago

Reply:
We could maybe add a TODO just to flag that? Nitpicking, that is not that relevant, but would spare us future work